letsencrypt/auth_handler.py

```diff
@@ -76,9 +76,16 @@ def get_authorizations(self, domains, best_effort=False):
 
```

Comment:
GitHub won't let me comment on the exact line, but the return type of this function should be updated. Looks like it's been wrong for a while...